ts/utils/openMaps.ts

```diff
@@ -0,0 +1,26 @@
+import { Linking, Platform } from "react-native";
+
+export function openMaps(
+  streetName: string,
+  latitude?: number,
+  longitude?: number
+) {
+  const intentScheme = Platform.select({
+    ios: "maps:0,0?q=",
+    android: "geo:0,0?q="
+  });
+  const latitudeLongitude = `${latitude},${longitude}`;
+  const label = streetName;
+  const url =
+    latitude && longitude
+      ? Platform.select({
+          ios: `${intentScheme}${label}@${latitudeLongitude}`,
+          android: `${intentScheme}${latitudeLongitude}(${label})`
+        })
+      : Platform.select({
+          ios: `${intentScheme}${label}`,
+          android: `${intentScheme}${label}`
+        });
+  // tslint:disable no-floating-promises
```

Comment:
Please handle the promise with a toast in case of failure.

Reply:
Done! [ecaf698](https://github.com/teamdigitale/io-app/pull/1174/commits/ecaf698f3c6147e58c015b02adff0fe31d10d9ec)